### ISSUE TITLE
Copycat cluster intermittently stuck in limbo

### ISSUE THREAD

**Issue by vyzo** [mediachain/mediachain#79]
The copycat cluster test may get stuck in limbo in the high-load CI testing environment when executing the mediachain testsuite.
An instance of the problem:
https://travis-ci.org/mediachain/mediachain/jobs/129933434

It appears that the cluster test is stuck with the second server joining the first in the second step in cluster formation.

**Comment by vyzo** [comment]
Investigating the trace, it appears that the initial member of the cluster fails to bootstrap and quietly dies.
Relevant logs from the trace:

```
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.catalyst.transport.NettyServer - Listening at /127.0.0.1:10004
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.ServerContext - /127.0.0.1:10004 - Transitioning to FOLLOWER
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.ServerContext - /127.0.0.1:10004 - Transitioning to CANDIDATE
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.CandidateState - /127.0.0.1:10004 - Starting election
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.ServerContext - /127.0.0.1:10004 - Transitioning to LEADER
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.ServerContext - /127.0.0.1:10004 - Found leader /127.0.0.1:10004

[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.state.ServerStateMachine - /127.0.0.1:10004 - Taking snapshot 1
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.copycat.server.CopycatServer - Server started successfully!

[copycat-server-/127.0.0.1:10005-copycat] INFO io.atomix.copycat.server.state.FollowerState - /127.0.0.1:10005 - Polling members [ServerMember[type=ACTIVE, status=AVAILABLE, serverAddress=/127.0.0.1:10004, clientAddress=null]]
[copycat-server-/127.0.0.1:10005-copycat] INFO io.atomix.copycat.server.state.ServerContext - /127.0.0.1:10005 - Found leader /127.0.0.1:10004
[copycat-server-/127.0.0.1:10004-copycat] WARN io.atomix.copycat.server.state.LeaderAppender - /127.0.0.1:10004 - Failed to configure /127.0.0.1:10005
[copycat-server-/127.0.0.1:10004-copycat] WARN io.atomix.copycat.server.state.LeaderAppender - /127.0.0.1:10004 - AppendRequest to /127.0.0.1:10005 failed. Reason: request timed out

[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.catalyst.transport.NettyClient - Connecting to /127.0.0.1:10005
[copycat-server-/127.0.0.1:10004-copycat] INFO io.atomix.catalyst.transport.NettyClient - Connecting to /127.0.0.1:10005
[copycat-server-/127.0.0.1:10004-copycat] WARN io.atomix.copycat.server.state.LeaderAppender - /127.0.0.1:10004 - AppendRequest to /127.0.0.1:10005 failed. Reason: request timed out

[copycat-server-/127.0.0.1:10005-copycat] INFO io.atomix.copycat.server.state.FollowerState - /127.0.0.1:10005 - Polling members [ServerMember[type=ACTIVE, status=AVAILABLE, serverAddress=/127.0.0.1:10004, clientAddress=/127.0.0.1:10004]]
...
```

**Comment by parkan** [comment]
I think we're good on this one? Feel free to reopen if not
